### PR TITLE
Update citation for "Initial conceptual demonstration..." paper

### PR DESCRIPTION
**Pull request by ryancoe**
## Description
Paper now published and available online; update DOI and hyperlinks

Fixes # (issue)
N/A

## Checklist:
N/A

### CODE REVIEW

File: examples/WaveBot/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 ```
 @Article{WecDesignOptimizationTool,
 	author = "Ryan G. Coe and Giorgio Bacelli and Sterling Olson and Vincent S. Neary and Mathew B. R. Topper",
-  	title  = {{Initial conceptual demonstration of control co-design for WEC optimization}},
+  	title  = {{Initial conceptual demonstration of control co-design for {WEC} optimization}},
   	year   = {2020},
-  	journal= {Journal of Ocean Engineering and Marine Energy (in press)},
-  	doi    = {10.36227/techrxiv.12928898}
+  	journal= {Journal of Ocean Engineering and Marine Energy},
+  	doi    = {10.1007/s40722-020-00181-9
```
Comment (H0R5E) [inline examples/WaveBot/README.md:12]: You've lost the closing bracket for the doi field.

**Review by H0R5E — CHANGES_REQUESTED** [review]

File: examples/WaveBot/README.md (hunk 1 above)
Comment (ryancoe) [reply examples/WaveBot/README.md]: fixed w. 45d67481a157e8dcc562cc299bad49f9f82cceba

File: examples/WaveBot/README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 This example was developed in conjuction with the following journal manuscript, which is currently under review.
-A preprint is avaialable at <https://doi.org/10.36227/techrxiv.12928898>.
+A preprint is avaialable at <https://doi.org/10.1007/s40722-020-00181-9>.
 
 ```
 @Article{WecDesignOptimizationTool,
 	author = "Ryan G. Coe and Giorgio Bacelli and Sterling Olson and Vincent S. Neary and Mathew B. R. Topper",
-  	title  = {{Initial conceptual demonstration of control co-design for WEC optimization}},
+  	title  = {{Initial conceptual demonstration of control co-design for {WEC} optimization}},
```
[the remark this answers is not in the document]
Comment (ryancoe) [reply examples/WaveBot/README.md:9]: in my experience, BibTex will not preserve upper case words unless you do this

[the remark this answers is not in the document]
Comment (H0R5E) [reply examples/WaveBot/README.md:9]:
```
@article{Coe2020initial,
	Author = {Ryan G. Coe and Giorgio Bacelli and Sterling Olson and Vincent S. Neary and Mathew B. R. Topper},
	Doi = {10.36227/techrxiv.12928898},
	Journal = {Journal of Ocean Engineering and Marine Energy (in press)},
	Title = {{INITIAL CONCEPTUAL DEMONSTRATION OF CONTROL CO-DESIGN FOR WEC OPTIMIZATION}},
	Year = {2020},
	Bdsk-Url-1 = {https://doi.org/10.36227/techrxiv.12928898}}
```

double_bracket

```
@article{Coe2020initial,
	Author = {Ryan G. Coe and Giorgio Bacelli and Sterling Olson and Vincent S. Neary and Mathew B. R. Topper},
	Doi = {10.36227/techrxiv.12928898},
	Journal = {Journal of Ocean Engineering and Marine Energy (in press)},
	Title = {INITIAL CONCEPTUAL DEMONSTRATION OF CONTROL CO-DESIGN FOR {WEC} OPTIMIZATION},
	Year = {2020},
	Bdsk-Url-1 = {https://doi.org/10.36227/techrxiv.12928898}}
```

single_bracket

[the remark this answers is not in the document]
Comment (ryancoe) [reply examples/WaveBot/README.md]: oh, I see what you meant, fixed w. ac7f622859cfc9c9abb14db7a78e72fdd643a188

**Review by H0R5E — APPROVED** [review]

**Comment by H0R5E** [comment]
Please go ahead and merge it.

**Comment by H0R5E** [comment]
NO WAIT! :octocat:

File: examples/WaveBot/README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # WaveBot
 
 This example was developed in conjuction with the following journal manuscript, which is currently under review.
-A preprint is avaialable at <https://doi.org/10.36227/techrxiv.12928898>.
+A preprint is avaialable at <https://doi.org/10.1007/s40722-020-00181-9>.
```
Comment (H0R5E) [inline examples/WaveBot/README.md:4]: This isn't a preprint any more...

**Review by H0R5E — CHANGES_REQUESTED** [review]

File: examples/WaveBot/README.md (hunk 3 above)
Comment (ryancoe) [reply examples/WaveBot/README.md]: updated w. 47d546b89bd9f010833e5d16856cfb5e9cd2393a

this is why we use PRs ;)

**Review by H0R5E — APPROVED** [review]

**Comment by H0R5E** [comment]
OK, well, now all that excitement is over, please merge it before I spot anything else.

**Pull request merged**